includes/minify/class-html.php

```
@@ -66,6 +74,14 @@ class HTML {
 		public function __construct( $html, $options ) {
 			$this->options = (array) $options;
 			$this->initialize_minification_settings();
+
+			// Cache delay JS exclusions to avoid redundant processing in loops.
+			$this->exclude_delay_js = array_merge(
+				array( 'wppo-lazyload', 'data-wppo-preserve' ),
+				Util::process_urls( $this->options['file_optimisation']['excludeDelayJS'] ?? array() )
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\PerformanceOptimise\Inc\Util' in method '__construct'. (undefined)

(StaticAccess)